frontend/app/src/composables/blockchain/index.ts

```diff
@@ -66,7 +66,7 @@ export const useBlockchains = () => {
     periodic = false
   ) => {
     const chain = get(blockchain);
-    await fetchAccounts(chain);
+    await fetchAccounts(chain, true);
```

Comment:
do we only call the method manually?

Reply:
for the refreshAccounts, no, sometimes it automatically, or can be triggered by refresh button.
Maybe I should rename it from `userInitiated` cause the purpose only to refresh the ens

Reply:
Okay, updated